src/BackgroundLayerPreview/BackgroundLayerPreview.tsx

```diff
@@ -19,7 +19,7 @@ export type BackgroundLayerPreviewProps = {
   width?: number;
   height?: number;
   layer: OlLayer;
-  activeLayer: OlLayer;
+  activeLayer?: OlLayer;
```

Comment:
Actually this would be a breaking change. Is this intended?

Reply:
ActiveLayer is not optional anymore, thank you!